flowey/flowey_lib_hvlite/src/artifact_openvmm_hcl_sizecheck.rs

```diff
@@ -48,3 +48,55 @@ pub mod publish {
         }
     }
 }
+
+/// Resolve the contents of an existing artifact.
+pub mod resolve {
+    use flowey::node::prelude::*;
+
+    #[derive(Serialize, Deserialize)]
+    pub struct OpenvmmSizeCheckArtifact {
+        pub bin: PathBuf,
+    }
+
+    flowey_request! {
+        pub struct Request {
+            pub openvmm_openhcl_x86: WriteVar<OpenvmmSizeCheckArtifact>,
+            pub artifact_dir: ReadVar<PathBuf>,
+        }
+    }
+
+    new_simple_flow_node!(struct Node);
+
+    impl SimpleFlowNode for Node {
+        type Request = Request;
+
+        fn imports(ctx: &mut ImportCtx<'_>) {
+            ctx.import::<flowey_lib_common::copy_to_artifact_dir::Node>();
+        }
+
+        fn process_request(request: Self::Request, ctx: &mut NodeCtx<'_>) -> anyhow::Result<()> {
+            let Request {
+                openvmm_openhcl_x86,
+                artifact_dir,
+            } = request;
+
+            ctx.emit_rust_step("resolve baseline openhcl artifact", |ctx| {
+                let openvmm_openhcl_x86 = openvmm_openhcl_x86.claim(ctx);
+                let artifact_dir = artifact_dir.claim(ctx);
+                move |rt| {
+                    let artifact_dir = rt.read(artifact_dir);
+
+                    let openvmm_openhcl_x86_output = OpenvmmSizeCheckArtifact {
+                        bin: artifact_dir.join("x64-openhcl-baseline").join("openhcl"),
```

Comment:
this doesn't seem quite right. at least - it doesn't line up with the format from the `publish` node above.

hmm, are you sure your artifact download node is returning the right directory? i.e: shouldn't it return the directory directly containing the artifact contents, as opposed to a directory containing _another_ directory, which then contains the artifact contents?